src/main/java/org/skriptlang/skript/bukkit/whitelist/elements/EvtPlayerWhitelist.java

```diff
@@ -0,0 +1,69 @@
+package org.skriptlang.skript.bukkit.whitelist.elements;
+
+import io.papermc.paper.event.server.WhitelistStateUpdateEvent;
+import io.papermc.paper.event.server.WhitelistStateUpdateEvent.WhitelistStatus;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptEvent;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.skript.registrations.EventValues;
+import ch.njol.util.Kleenean;
+
+public class EvtPlayerWhitelist extends SkriptEvent {
+
+	static {
+		Skript.registerEvent("Player Whitelist", EvtPlayerWhitelist.class, WhitelistStateUpdateEvent.class,
+				"player whitelist [state] (change[d]|toggle[d]|update[d])",
```

Comment:
```suggestion
				"player whitelist [state] (change[d]|update[d])",
```
the toggled syntax sounds like i'm turning the whitelist on or off, not updating a single player.